eng/scripts/Set-VcpkgWriteModeCache.ps1

```diff
@@ -1,3 +1,5 @@
+#cspell: ignore azblob
```

Comment:
Since azblob shows up several times, would it be better to put it inthe global dictionary?

Reply:
Thank you! Makes sense. Done.